src/FullBuild/IO.fs

```diff
@@ -14,14 +14,23 @@
 
 module IO
 
+open System
+
 type Result =
     { Code: int
       Info : string
       Out: string list
       Error: string list }
 
 let private resultToError execResult =
-    if execResult.Code <> 0 then (sprintf "Operation '%s' failed with error %d" execResult.Info execResult.Code) |> Some
+    if execResult.Code <> 0 then 
+        let prefix p = function [] -> [] | l -> p :: l
+
+        sprintf "Operation '%s' failed with error %d" execResult.Info execResult.Code 
+        :: prefix "Out:" execResult.Out
+        @ prefix "Error:" execResult.Error
+        |> String.concat Environment.NewLine
```

Comment:
|> must be indented

Reply:
This is standard indentation for |>...

Reply:
For me it's the same as everywhere in the full-build, so ok

Reply:
Yes it's strange and indentation is still not ok.

Reply:
It's compiling and aligned on previous lines, I don't see what you expect ?